x-pack/plugins/apm/public/components/app/service_node_overview/index.tsx

```diff
@@ -141,6 +159,7 @@ function ServiceNodeOverview() {
         defaultMessage: 'Thread count max',
       }),
       field: 'threadCount',
+      dataType: 'number',
```

Comment:
Thanks for adding this. Is that to right-align these values?

Reply:
Yes, other tables have the number values aligned to the right.

Reply:
@sqren I gave this feedback to @MiriamAparicio seeing as we've right-aligned most of our other tables to ensure that numbers are displayed with better legibility.

Reply:
Makes sense 👍